bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/LimitsListener.java

```diff
@@ -160,8 +238,25 @@ public void propertyChange(PropertyChangeEvent evt) {
 					mm.addMarker(false, this, Severity.Warning, "Target comes within 1\u00B0 of upper HA constraint.", v, a);
 				}
 
-				break;
+				if (minHA < haLimitMin && maxHA > haLimitMax && elevationIntervals != null && elevationIntervals.hasNext()) {
```

Comment:
Hmm, aren't these conditions already accounted for above in the errors and warnings.  That is, shouldn't you only add this information if `minHA >= haLimitMin && maxHA <= haLimitMax`?